packages/file-type-icons/src/FileTypeIconMap.ts

```diff
@@ -406,7 +404,6 @@ export const FileTypeIconMap: { [key: string]: { extensions?: string[] } } = {
       'ithmb',
       'jar',
       'kb',
-      'kdc',
```

Comment:
Same here, is this a dupe? I'm not familiar enough with the nuances to know, but, I don't see another `kdc` in here.

Reply:
kdc is used as both "raw" image format by some kodak cameras, and also by Kasperski antivirus as virus defs stored somewhere in the OS's system folders. since they're both real, and OD is about end user storage, we are leaving the image entry in place and removing the sysfile entry